orocos-kdl-python/PKGBUILD

```diff
@@ -34,6 +34,7 @@ build() {
   cd "${srcdir}/${_dir}-${pkgver}/${_pkgname}"
   mkdir -p build && cd build
   cmake -DCMAKE_INSTALL_PREFIX=/usr \
+        -DCMAKE_BUILD_TYPE=Release \
```

Comment:
We should not be using the Release mode see: https://wiki.archlinux.org/title/CMake_package_guidelines#CMake_can_automatically_override_the_default_compiler_optimization_flag